public/js/gulliver.es6.js

```diff
@@ -164,6 +188,33 @@ class Gulliver {
       window.history.back();
     });
   }
+
+ /**
+  * Setup/configure search button
+  */
+  setupSearchElements() {
```

Comment:
Sounds like we could create a `search-button.js`, encapsulate this code there and just instantiate it from `gulliver.js`